api-module-library/microsoft-teams/manager.js

```diff
@@ -32,7 +32,7 @@ class Manager extends ModuleManager {
             client_secret: process.env.TEAMS_CLIENT_SECRET,
             redirect_uri: `${process.env.REDIRECT_URI}/microsoft-teams`,
             tenant_id: process.env.TENANT_ID,
-            team_id: process.env.TEAMS_ID,
+            team_id: process.env.TEAMS_TEAM_ID,
```

Comment:
This one, I think we discussed on a call today, would be provided by the user on prompting, so it would be external from the API class (would have to be an argument passed in... now I see the "get members" point you made on call, get members is team-focused, iterating through _all_ teams and corresponding members might be a lot but also may be the only way to get the actual IDs that the bot framework needs (as using the Graph API doesn't get us the right User IDs, is that right?)